README.md

```diff
@@ -56,6 +56,9 @@ The project ID of the TestRail project.
 ##### `suite_id` (**Required**)
 The suite ID of the TestRail project. Does not include "S" prefix.
 
+##### `exclude_e2e` (**Optional**)
+Used in projects that don't require nightwatch testing like in EDS. When true will close milestones regardless of E2E reports.
+
```

Comment:
If your goal is to make it so that you can have the milestone be closed at the end of a test run, why not introduce a parameter for closing the milestone at the end of the testrun? Such as `closeMilestone`.

This current param might be lost in the cracks.

Reply:
I don't want to just close the milestone at the end of a test run, take this pipeline as an example, lets say `build / test-unit` step passes with success, the report file will get created, a milestone and a test run will be opened but they won't get closed, and that is correct as we don't want to close the milestone yet considering the following step `build / code-analysis` may fail, in that case a new commit will need to be pushed and `build / test-unit` executed again, if the milestone is closed a new milestone will be created instead of using the existing one, we don't want that correct? @Q4Yilong maybe you can help me here, I don't know if I was clear with this explanation, feel free to elaborate. I think we want to make sure the milestone gets closed only when the whole pipeline has been finished.
![image](https://github.com/q4mobile/stratus-testrail-reporter/assets/7043013/9aaba339-3e48-4732-8766-0cb34e97c8bc)

Reply:
@vitalizatroutine I think the key here is the environment. do you know when we should close the milestone? 
we have unit test that runs stratus during the `build` phase. not sure if we should close the milestone only when the code is successfully deployed to prod.

Reply:
Your milestone should be closed when your testing has completed in production, as closing the milestone signifies that something if available and tested in production.

```
environment === Environment.Production && (suite.name.includes("E2E") || closeMilestone)

Reply:
Parameter name was updated from `exclude_e2e` to `close_milestone` in https://github.com/q4mobile/stratus-testrail-reporter/pull/32/commits/62132af5d33827aa78a4b6acf1308985b2b57306.